docs/howto/logging-to-a-file/index.adoc

```diff
@@ -44,14 +44,14 @@ Under `logging`, add a `periodic-rotating-file-handlers` section with a new hand
 periodic-rotating-file-handlers:
```

Comment:
Comment to line 27: adding a formatter isn't strictly necessary, there are two pre-defined formatters available. One called `PATTERN`, suitable for logging to a file, and `COLOR_PATTERN`, suitable for logging to an interactive console. Showing how to create your own pattern is certainly desirable, though, so I wouldn't remove that -- I'd just mention that there are 2 pre-defined patterns available (or 1, really, when it comes to logging to a file).

Reply:
OK, will do, thanks.